### PR TITLE
Instance comment 

### PR DESCRIPTION
**Pull request by msn1992**

### CODE REVIEW

File: src/main/webapp/app/instance/instance-comment/instance-comment.component.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
+import { Component, Input, ViewChild, OnInit } from '@angular/core';
+import 'leaflet/dist/leaflet.css';
```
Comment (hpardess) [inline src/main/webapp/app/instance/instance-comment/instance-comment.component.ts]: what is the use of this library to be imported

File: src/main/webapp/app/dynamic-form/geopoint-question/geopoint-question.component.ts [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+import 'leaflet/dist/leaflet.css';
+import {MatDatepickerInputEvent} from '@angular/material/datepicker';
+
+@Component({
+  selector: 'geopoint-question',
+	templateUrl: './geopoint-question.component.html',
+	styleUrls: ['./geopoint-question.component.scss'],
+})
+export class GeopointQuestionComponent implements OnInit {
+	@Input() question: any;
```
Comment (hpardess) [inline src/main/webapp/app/dynamic-form/geopoint-question/geopoint-question.component.ts]: Do not commit two issues task to same branch. Keep the code of each issue seperated.

File: src/main/java/com/nsia/service/InstanceCommentService.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
+package com.nsia.service;
+
+import com.nsia.model.InstanceComment;
+
+public interface InstanceCommentService {
+	public InstanceComment create(InstanceComment InstanceComment);
+    public InstanceComment delete(Long id);
+    public InstanceComment findById(Long id);
+    public InstanceComment findAll(Long instanceId);
```
Comment (hpardess) [inline src/main/java/com/nsia/service/InstanceCommentService.java]: by its name it clearly says that it returns a collection, not a single object.

File: src/main/java/com/nsia/model/InstanceComment.java [hunk 4]
```diff
@@ -6,11 +6,110 @@
...
 public class InstanceComment {
-
+ 
 	@Id
 	@GeneratedValue(strategy = GenerationType.AUTO)
 	private Long id;
-}
+ 
+	@Column(name = "name")
+	private String name;
```
Comment (hpardess) [inline src/main/java/com/nsia/model/InstanceComment.java]: what is the purpose of this field in instance_comment table?

File: src/main/webapp/app/instance/instance-comment/instance-comment.component.ts (hunk 1 above)
Comment (msn1992) [reply src/main/webapp/app/instance/instance-comment/instance-comment.component.ts]: ViewChild and leaflet are using for the map. there is no use of that.

File: src/main/java/com/nsia/model/InstanceComment.java (hunk 4 above)
Comment (msn1992) [reply src/main/java/com/nsia/model/InstanceComment.java]: @hpardess this component was in progress and was not completed. as you told I commit it sorry for that. if you got much confused, leave it I will complete it.

**Pull request closed without merging**